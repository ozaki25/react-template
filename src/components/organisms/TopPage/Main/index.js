import React from 'react';
import styled from 'styled-components';

import BasicText from '../../../atoms/BasicText';
import Link from '../../../atoms/Link';
import StartReissueButton from '../../../molecules/StartReissueButton';

function Main() {
  return (
    <Container>
      <StartReissueButtonContainer>
        <StartReissueButton href="/" />
      </StartReissueButtonContainer>
      <BasicText>※今後、本アプリでできる手続を順次拡大していく予定です。</BasicText>
      <LinkContainer>
        <Link href="#">ライセンスについて</Link>
      </LinkContainer>
    </Container>
  );
}

export const Container = styled.section`
  padding: 15px 10px 20px;
`;

export const StartReissueButtonContainer = styled.div`
  text-align: center;
`;

export const LinkContainer = styled.div`
  text-align: right;
`;

export default Main;
