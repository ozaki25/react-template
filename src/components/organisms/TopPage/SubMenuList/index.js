import React from 'react';
import styled from 'styled-components';
import MenuButton from 'src/components/atoms/buttons/MenuButton';

function SubMenuList() {
  return (
    <Container>
      <MenuButtonContainer>
        <MenuButton onClick={() => console.log('click')}>申込履歴の確認</MenuButton>
      </MenuButtonContainer>
      <MenuButtonContainer>
        <MenuButton onClick={() => console.log('click')}>よくあるご質問</MenuButton>
      </MenuButtonContainer>
      <MenuButtonContainer>
        <MenuButton onClick={() => console.log('click')}>残高照会・振込等</MenuButton>
      </MenuButtonContainer>
    </Container>
  );
}

export const Container = styled.ul`
  margin: 0;
  padding: 0;
`;

export const MenuButtonContainer = styled.li`
  margin-bottom: 10px;
  list-style-type: none;
`;

SubMenuList.propTypes = {};

SubMenuList.defaultProps = {};

export default SubMenuList;
