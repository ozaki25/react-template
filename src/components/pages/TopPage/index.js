import React from 'react';
import Main from '../../organisms/TopPage/Main';

function TopPage() {
  return (
    <React.Fragment>
      <header>アプリメニュー</header>
      <Main />
      <footer>ver.1.0.1</footer>
    </React.Fragment>
  );
}

export default TopPage;
