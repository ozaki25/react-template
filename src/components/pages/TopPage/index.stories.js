import React from 'react';

import { storiesOf } from '@storybook/react';
import { withInfo } from '@storybook/addon-info';

import TopPage from '.';

const stories = storiesOf('pages', module);

stories.add('TopPage', withInfo('TopPage')(() => <TopPage />));
