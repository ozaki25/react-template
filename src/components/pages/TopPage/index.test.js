import React from 'react';
import { snapshot } from '../../../../test/helpers';
import TopPage from './';

snapshot('TopPage', <TopPage />);
