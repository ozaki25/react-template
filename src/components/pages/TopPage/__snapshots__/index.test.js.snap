// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot: TopPage 1`] = `
<React.Fragment>
  <header>
    アプリメニュー
  </header>
  <Main />
  <footer>
    ver.1.0.1
  </footer>
</React.Fragment>
`;
